tools/Install/setup.sh

```diff
@@ -138,7 +138,7 @@ fi
 
 XMOS_TAG=$2
 
-shift 1
+shift 2
```

Comment:
Why do we process non-option arguments before option arguments? This is the opposite of what everyone else does, ever! ;-)

Reply:
This is similar to what is done in the Amazon version of the setup file:
https://github.com/alexa/avs-device-sdk/blob/master/tools/Install/setup.sh
What do you propose to change exactly?